src/main/java/com/rybka/Application.java

```diff
@@ -30,10 +34,25 @@ public static void main(String[] args) {
         var reader = new PropertyReader(PropertyInfo.PROPERTY_FILE_NAME).getProperties();
         var connector = exchangeSourceMap.get(reader.getProperty(PropertyInfo.PROPERTY_EXCHANGE_SOURCE));
 
+        var exportFolder = reader.getProperty(PropertyInfo.PROPERTY_EXPORT_FOLDER);
+        var exportFileName = FileUtils.generateFileName(reader.getProperty(PropertyInfo.PROPERTY_EXPORT_TYPE));
+
+        var consoleExportService = new ConsoleExportService();
+        var csvExportService = new CSVExportService(new CsvMapper(), exportFolder, exportFileName);
+        var jsonExportService = new JSONExportService(objectMapper, exportFolder, exportFileName);
+
+        var exportConfigMap = Map.of(
+                PropertyInfo.PROPERTY_EXPORT_CONSOLE, consoleExportService,
+                PropertyInfo.PROPERTY_EXPORT_CSV, csvExportService,
+                PropertyInfo.PROPERTY_EXPORT_JSON, jsonExportService);
+
+        var exportService = exportConfigMap.get(reader.getProperty(PropertyInfo.PROPERTY_EXPORT_TYPE));
```

Comment:
Handle situation when user enter `txt` value. As far as we don't value handler for that we should throw an exception.

Reply:
Do the same for the connectors map

Reply:
fixed